### PR TITLE
Update glide dependencies

### PR DESCRIPTION
**Pull request by ruflin**
* Change package in glide.yml to beats instead of libbeat
 * Add glide.lock to .gitignore file
 * Add cloudfoundry dependency as it was missing
 * Fix golang.org/x/text version with commit id as newest version is incompatible
 * Update dependencies of the following packages to the most recent version
   * github.com/garyburd/redigo
   * github.com/stretchre/testify
   * gopkg.in/yaml.v2
   * github.com/elastic/gosigar
   * github.com/tsg/gopacket
   * github.com/pmezard/go-difflib
   * golang.org/x/sys

### CODE REVIEW

**Comment by andrewkroh** [comment]
Why do we have cloudfoundary dependency? After pulling in the updated elastic/gosigar examples there shouldn't be a dependencies on cloudfoundary. Maybe I am just not seeing the dep...?

**Comment by ruflin** [comment]
There still seem to be two dependencies: One in fakes/fake_sigar.go and sigar_windows_test.go. I agree that it is probably not used during compiling but it is still detected by glide as a dependency. We should probably remove / cleanup those files in our gosigar lib?

**Comment by monicasarbu** [comment]
I can take care of this.

**Comment by monicasarbu** [comment]
Fixed by https://github.com/elastic/gosigar/pull/10

**Comment by ruflin** [comment]
Based on https://github.com/elastic/gosigar/pull/10 I updated gosigar again and removed the cloudfoundry dependency.

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 # Files
 .DS_Store
+/glide.lock
+/beats.iml
```
Comment (tsg) [inline .gitignore]: Just curious, what's this file?

Comment (ruflin) [inline .gitignore]: This files is coming from the newest version of IntellJ. It has the module config inside. I tried to remove it but then IntellJ does not show any go files anymore :-(

**Pull request merged**